classes/tree/Tree.php

```diff
@@ -101,7 +101,7 @@ public function setAttribute($name, $value)
 
     public function getAttribute($name)
     {
-        return $this->hasAttribute($name) ? $this->_attributes[$name] : null;
+        return $this->_attributes[$name] ?: null;
```

Comment:
```suggestion
        return $this->_attributes[$name] ?? null;
```
If the index is not defined, it will raise an error, you're looking for this: https://stitcher.io/blog/shorthand-comparisons-in-php#null-coalescing-on-arrays